base/src/main/java/com/smartdevicelink/proxy/rpc/BodyInformation.java

```diff
@@ -1,215 +1,423 @@
-/*
- * Copyright (c) 2017 - 2019, SmartDeviceLink Consortium, Inc.

```

Comment:
@kostyaBoss It looks like you used the code generator tool to re-generate the entire file.  Can you just merge in the changes that are specific to this PR (in the HTML these would be the parameters from `driverDoorAjar` to `roofStatuses`)?